spec/collectionspace/converter/publicart/acquisition_spec.rb

```diff
@@ -112,48 +87,27 @@
 
   describe 'map_publicart' do
     pa = 'acquisitions_publicart'
-    context 'publicart fields' do
+    context 'non-authority/vocab fields' do
       [
         "/document/#{pa}/accessionDate",
-        "/document/#{pa}/acquisitionDate",
-        "/document/#{pa}/acquisitionDate",
+        "/document/#{pa}/acquisitionDates/acquisitionDate",
       ].each do |xpath|
         context "#{xpath}" do
-          it 'all values will be URNs' do
-            expect(urn_values(doc, xpath)).not_to include('Not a URN')
+          let(:doctext) { get_text(doc, xpath) }
+          it 'is not empty' do
+            expect(doctext).to_not be_empty
           end
```

Comment:
@Paulmulonzia - These date fields should not be URNs! These tests definitely shouldn't have been passing, and are what alerted me to the fact of my test bug.